### PR TITLE
[BUG] improve code in EnginePluginRestful

### PR DESCRIPTION
**Pull request by GuoPhilipse**
### What is the purpose of the change

improve code in EnginePluginRestful

### Related issues/PRs

Related issues: #3911
Related pr:#3913

### Brief change log

- 1、fix typos ;
- 2、fix unused code

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/server/restful/EnginePluginRestful.java [hunk 1]
```diff
@@ -237,7 +235,7 @@ public Message uploadEnginePluginBML(
         }
         engineConnResourceService.refreshAll(true, false);
         log.info("{} finished to upload enginePlugin", username);
-        return Message.ok().data("mes", "upload file success");
+        return Message.ok().data("msg", "upload file success");
```
Comment (wForget) [inline linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/server/restful/EnginePluginRestful.java:238]: This key will be used in linkis-web:

https://github.com/apache/incubator-linkis/blob/8fbdc3abb68b302d2ba97157abd4d283abe86574/linkis-web/src/apps/linkis/module/EnginePluginManagement/index.vue#L340

Comment (GuoPhilipse) [reply linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/server/restful/EnginePluginRestful.java:238]: thanks @wForget  for your review, will  address the code later.

**Comment by wForget** [comment]
There seems to be an error here. Could you please fix it together?

https://github.com/apache/incubator-linkis/blob/c9b1a0572fe7877461cbc073d9e2e1018a6ef2ec/linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/server/restful/EnginePluginRestful.java#L171

`uploadEnginePluginBML` to `updateEnginePluginBML`

**Comment by wForget** [comment]
There is also an error here caused by #3895.

https://github.com/apache/incubator-linkis/blob/8fbdc3abb68b302d2ba97157abd4d283abe86574/linkis-web/src/apps/linkis/module/EnginePluginManagement/index.vue#L354

**Comment by GuoPhilipse** [comment]
> There seems to be an error here. Could you please fix it together?
> 
> https://github.com/apache/incubator-linkis/blob/c9b1a0572fe7877461cbc073d9e2e1018a6ef2ec/linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/server/restful/EnginePluginRestful.java#L171
> 
> `uploadEnginePluginBML` to `updateEnginePluginBML`

fine for me ,will take a look later today~

**Review by wForget — APPROVED** [review]
LGTM, Thanks

**Review by casionone — APPROVED** [review]

**Pull request merged**